src/otx/algo/segmentation/litehrnet.py

```diff
@@ -87,7 +87,7 @@ def _exporter(self) -> OTXModelExporter:
     @property
     def ignore_scope(self) -> dict[str, Any]:
         """Get the ignored scope for LiteHRNet."""
-        if self.model_name == "large":
+        if self.model_version == "lite_hrnet_x":
```

Comment:
It should be model_name, not the model_version. I think it is the reason why tests are failed.

Reply:
Thanks for letting us know. I think we should close this PR and mergeback to rc1 for now.